m4_sessions/1711-inning/Group_3.md

```diff
@@ -8,11 +8,13 @@ Session Outcomes: What will students leave with or gain from this session? A bet
 
 Session Outline: Include the flow of the session and the time for each section.
 
-15 mins Icebreaker/intros
-10 mins article summary of open source ideas and problems
-15 mins discussion
-15 mins how to contribute
-10 mins the future of open source
-10 mins closing and call to action
+15 mins Icebreaker/intros. Eddie will lead this section. "Do you have a nickname?"
+5 min video on what is open source? Will add content. Currently we have a bunch of videos that could work. We need to sort out which one to use. We'll have that done by EOD Monday.
+[7 mins article on licensing and license differences.](https://lukasa.co.uk/2012/05/GPL_vs_MIT_Which_License_To_Use/) Ash will lead this topic.
```

Comment:
Are they just reading it?  Do you want them to be thinking of something while they are reading?